webview/src/components/file-input.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+  import { Add } from "svelte-codicons"
+  export let id: string
+  export let required: boolean = false
+  type FileEventTarget = EventTarget & { files: FileList }
+
+  const defaultText = "No file chosen"
+
+  function handleOnChange(e: Event) {
+    const target = e.target as FileEventTarget
+    const fileChosen = document.getElementById(`${id}-file-chosen`)
```

Comment:
I don't think this direct dom access is considered good practice nowadays.
Unless I'm misinformed (granted I haven't done web development in a while), you should use svelte's bindings to re-render the component.

Reply:
In order to change the default text for the file input we needed to directly modify the `textContent` for the element. It's not something that would change base on a re-render. 

I've removed this custom `FileInput` in in favor of just using CSS styling for now. Only difference is it does not include the icon that was in the designs. Maybe in the future we can revisit creating a fully accessible and custom file input with some nice icons ✨ but perhaps more of a lift than what's needed at the moment.

<img width="388" alt="Screen Shot 2022-09-30 at 3 55 04 PM" src="https://user-images.githubusercontent.com/15069288/193361451-3f1c452a-e506-4a75-8e40-6a20c112df47.png">